### ISSUE TITLE
Errors during startup of vim python file editing ... Happens after an update of Anaconda //Thx

### ISSUE THREAD

**Issue by ysatGitHub** [klen/python-mode#634]
Error detected while processing /Users/YAOSHUN/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
Traceback (most recent call last):
Press ENTER or type command to continue
Error detected while processing /Users/YAOSHUN/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "<string>", line 1, in <module>
Press ENTER or type command to continue
Error detected while processing /Users/YAOSHUN/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "/Users/YAOSHUN/.vim/bundle/python-mode/pymode/lint.py", line 9, in <module>
Press ENTER or type command to continue
Error detected while processing /Users/YAOSHUN/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
    from pylama.lint.extensions import LINTERS
Press ENTER or type command to continue
Error detected while processing /Users/YAOSHUN/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "/Users/YAOSHUN/.vim/bundle/python-mode/pymode/libs/pylama/lint/extensions.py", line 30, in <module>
Press ENTER or type command to continue
Error detected while processing /Users/YAOSHUN/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
    from pkg_resources import iter_entry_points
Press ENTER or type command to continue
Error detected while processing /Users/YAOSHUN/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "/Users/YAOSHUN/.vim/bundle/python-mode/pymode/libs/pkg_resources/__init__.py", line 21, in <module>
Press ENTER or type command to continue
Error detected while processing /Users/YAOSHUN/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
    import io
Press ENTER or type command to continue
Error detected while processing /Users/YAOSHUN/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "/Users/YAOSHUN/Anaconda/lib/python2.7/io.py", line 51, in <module>
Press ENTER or type command to continue
Error detected while processing /Users/YAOSHUN/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
    import _io
Press ENTER or type command to continue
Error detected while processing /Users/YAOSHUN/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
ImportError: dlopen(/Users/YAOSHUN/Anaconda/lib/python2.7/lib-dynload/_io.so, 2): Symbol not found: __PyCodecInfo_GetIncrementalDecoder

**Comment by chriscampbell19** [comment]
I have a very similar error message but it goes on slightly longer
```
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
Traceback (most recent call last):
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "<string>", line 1, in <module>
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "/Users/chrisCampbell/.vim/bundle/python-mode/pymode/lint.py", line 9, in <module>
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
    from pylama.lint.extensions import LINTERS
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "/Users/chrisCampbell/.vim/bundle/python-mode/pymode/libs/pylama/lint/extensions.py", line 30, in <module>
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
    from pkg_resources import iter_entry_points
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "/Users/chrisCampbell/.vim/bundle/python-mode/pymode/libs/pkg_resources/__init__.py", line 21, in <module>
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
    import io
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/io.py", line 51, in <module>
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
    import _io
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
ImportError: dlopen(/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload/_io.so, 2):
 Symbol not found: __PyCodecInfo_GetIncrementalDecoder
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  Referenced from: /opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload/_io.so
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  Expected in: flat namespace
Error detected while processing /Users/chrisCampbell/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
 in /opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload/_io.so
```

I haven't knowingly updated anything in the past few days so I am confused by this.

My vim is compiled with python 2.7 as well which was a suggestion I found in another issue thread.

**Comment by outcoldman** [comment]
I had the same issue, but successfully fixed. 
In my case I compiled python and vim with homebrew, when PYTHON_PATH has been specified and set to one of my dev environments, where I also had some libraries, including io. 
Workaround was simple:
open new terminal, make sure that you do not have custom PYTHON_PATH, uninstall python, uninstall vim. Reinstall both of them.

**Comment by ysatGitHub** [comment]
Thanks, outcoldman.

What do you think the problem is? The custom PYTHON_PATH? Thanks.

**Comment by ysatGitHub** [comment]
Problem solved.

Culprit is the update from python 2.7.10 to 2.7.11.

If you are using conda package control, simply run "conda install python=2.7.10" will solve this problem.

At lease it works for me.

**Comment by chriscampbell19** [comment]
Thanks @ysatGitHub, under macports this is fixed with 'sudo port activate python27 @2.7.10_3' where the version number depends on what you have installed already.

**Comment by Arieg419** [comment]
Kudos to @F1LT3R 's solution, since it worked for me as well.

I did the following:

1. http://macvim-dev.github.io/macvim/ (Download the latest binary)
2. Open the MacVim.dmg file.
3. Drag it onto Applications, overwriting the MacVim you previously had.

Fixed it for me!

Cheers :)
